src/jira/__tests__/e2e.jira.test.ts

```diff
@@ -0,0 +1,98 @@
+import { IMerjoonProjects, IMerjoonTasks, IMerjoonUsers } from "../../common/types";
+import { ID_REGEX } from "../../utils/regex";
+import { getJiraService } from "../jira-service";
+import { JiraService } from "../service";
+
+describe("Jira ", () => {
```

Comment:
let's add one more common test checkReferences which will make sure that ids in task.assignees and task.projects are actually correct user.ids and project.ids